internal/graph/signals_gen.resolvers.tmpl

```diff
@@ -0,0 +1,30 @@
+// Code generated  with `make gql-model` DO NOT EDIT.
+package graph
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/DIMO-Network/telemetry-api/internal/graph/model"
+)
+
+{{ range .Signals }}
+// {{ GQLGenResolverName .JSONName }} is the resolver for the {{ .JSONName }}
+{{- if eq .GQLType "Float" }}
+func (r *signalAggregationsResolver) {{ GQLGenResolverName .JSONName }}(ctx context.Context, obj *model.SignalAggregations, agg model.FloatAggregation) (*float64, error) {
+    vn, ok := obj.ValueNumbers[model.AliasKey{Name: "{{ .JSONName }}", Agg: agg.String()}]
```

Comment:
This could probably be consolidated more.

Reply:
Yeah but I don't mind verbose template code. As long as it doesn't decrease the readability.